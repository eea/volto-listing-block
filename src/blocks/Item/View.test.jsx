import React from 'react';
import { Provider } from 'react-intl-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import ViewItemBlock from './View';

const mockStore = configureStore();

const store = mockStore({
  content: {
    data: {
      id: 'test',
      placeholder: 'placeholder',
    },
  },
  intl: {
    locale: 'en',
    messages: {},
  },
});

const mockData = {
  data: {
    id: 'test',
    placeholder: 'placeholder',
  },
};

describe('ViewItemBlock', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <Provider store={store}>
        <ViewItemBlock data={mockData} />
      </Provider>,
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
