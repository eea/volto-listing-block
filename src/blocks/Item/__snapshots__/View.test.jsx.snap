// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewItemBlock renders correctly 1`] = `
<div
  className="ui unstackable items row"
>
  <div
    className="item"
  >
    <div
      className="content"
    />
  </div>
</div>
`;
